src/components/HomePage/HomePage.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { handleGetWishlists } from '../../client/client';
+import HomePage from './HomePage';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { Accordion, Fab } from '@material-ui/core/';
+
+describe('HomePage', () => {
+  let component;
+  const mockHandleGetWishlists = jest.fn();
+
+  beforeEach(() => {
+    mockHandleGetWishlists.mockImplementation(handleGetWishlists);
+    component = mount(
+      <MemoryRouter>
+        <HomePage handleGetWishlists={mockHandleGetWishlists} />
+      </MemoryRouter>
+    );
+  });
+
+  afterEach(() => {
+    mockHandleGetWishlists.mockReset();
+  });
+
+  it('should load wishlists', () => {
+    expect(mockHandleGetWishlists).toBeCalled();
+  });
+});
```

Comment:
It might be worth considering testing failure states as well - how would we expect the HomePage to behave in the case of different errors occurring, what would it display or not display?